test/Microsoft.ML.Sweeper.Tests/Microsoft.ML.Sweeper.Tests.csproj

```diff
@@ -1,10 +1,12 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <DefineConstants>CORECLR</DefineConstants>
+    <DefineConstants>CORECLR;PORTED</DefineConstants>
```

Comment:
I'm not sure this `PORTED` buys us anything.  Can we just leave the test code out until it is ready?

Reply:
i introduced it so that we don't forget in future to port those tests in the future when the class is ported

Reply:
Part of me want to reference this test files in our internal repo, and just set Ported to true, so we can run them internally, but considering what we don't reference internally any tests files right now, I'm fine with just cutting them off.

Reply:
You can add a link to the tests in #733 and say "when we port that component, make sure these 3 tests are ported as well".  That is much cleaner than having this dead code in our repo.  If the component is never ported, then we don't have to deal with this dead code.